pkg/kubelet/kubelet_node_status.go

```diff
@@ -463,7 +463,7 @@ func (kl *Kubelet) tryUpdateNodeStatus(tryNumber int) error {
 	// If update finishes successfully, mark the volumeInUse as reportedInUse to indicate
 	// those volumes are already updated in the node's status
 	kl.volumeManager.MarkVolumesAsReportedInUse(updatedNode.Status.VolumesInUse)
-	return nil
+	return kl.containerManager.UpdateNodeAllocatableCgroups(node.Status.Capacity)
```

Comment:
Ref. comment above, this will turn into something like (again, names may have to change):

```go
if hasChanged := UpdateNodeAllocatableCgroups(updatedNode.Status.Capacity); hasChanged {
    if err := cm.enforceNodeAllocatableCgroups(); err != nil {
        return fmt.Errorf("unable to update cgroups: %+v", err);
    }
    return nil
}
```

or maybe:

```go
if shouldUpdate := shouldUpdate(updatedNode.Status.Capacity); shouldUpdate {
    if err := cm.UpdateAndEnforceNodeAllocatableCgroups(node.Status.Capacity); err != nil {
        return fmt.Errorf("unable to update cgroups: %+v", err);
    }
    return nil
}
```

Any thoughts?